src/types/UserProps.ts

```diff
@@ -1,4 +1,5 @@
 interface UserProps {
+    recipientUserId?: string;
```

Comment:
Why is there a recipientUserId and an `id` on a User? Isn't the recipient a user with the role `Recipient`?